packages/wallet/frontend/src/pages/auth/signup.tsx

```diff
@@ -32,7 +32,7 @@ const SignUpPage: NextPageWithLayout = () => {
               openDialog(
                 <SuccessDialog
                   onClose={closeDialog}
-                  content="Your account was created."
+                  content="Welcome to Interledger Testnet. Please verify your email address."
```

Comment:
We should provide clearer instructions about the next steps for the user. It might be helpful to display a message indicating that an email has been sent to verify the email address and guide them on what to do next.